dolphinscheduler-data-quality/src/main/java/org/apache/dolphinscheduler/data/quality/utils/ConfigUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.data.quality.utils;
+
+import org.apache.dolphinscheduler.data.quality.config.Config;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ConfigUtils {
+
+    private ConfigUtils() {
+        throw new IllegalStateException("Construct ConfigUtils");
+    }
+
+    /**
+     * Extract sub config with fixed prefix
+     *
+     * @param source config source
+     * @param prefix config prefix
+     * @param keepPrefix true if keep prefix
+     */
+    public static Config extractSubConfig(Config source, String prefix, boolean keepPrefix) {
+        Map<String, Object> values = new LinkedHashMap<>();
+
+        for (Map.Entry<String, Object> entry : source.entrySet()) {
+            final String key = entry.getKey();
+            final String value = String.valueOf(entry.getValue());
+
+            if (key.startsWith(prefix)) {
+                if (keepPrefix) {
+                    values.put(key, value);
+                } else {
+                    values.put(key.substring(prefix.length()), value);
+                }
+            }
+        }
+
+        return new Config(values);
+    }
```

Comment:
This method is only used in 2 places and the method can be written in a one-line statement, maybe just inline it and not to provide too many util classes?

```java
               new Config(source.entrySet()
                                .stream()
                                .filter(it -> it.getKey().startsWith(prefix))
                                .collect(toMap(it -> keepPrefix ? it.getKey() : it.getKey().substring(prefix.length()), Entry::getValue)));
```

Reply:
This utils will be used more frequently with the increase of readers